x-pack/plugins/serverless_search/kibana.jsonc

```diff
@@ -15,6 +15,7 @@
     "requiredPlugins": [
       "cloud",
       "console",
+      "data",
```

Comment:
Could use use the DataViews plugin directly instead?

Reply:
Question:  is there a reason why using DataViews plugin is recommended?

Reply:
Is data-plugin depreciated?

Reply:
No, just its re-export of the data view plugin

Reply:
Makes sense thanks!

Reply:
Added in 801bd5d3aceea4e9ed78d8c984afaa7a06ff52e9